docs/modules/serialization/pages/serialization.adoc

```diff
@@ -86,13 +105,8 @@ may incur costs on short-lived use cases
 
 |Java only
 
-| xref:implementing-dataserializable.adoc[`DataSerializable`]
-| * More CPU and memory usage efficient than `Serializable`
-| * Specific to Hazelcast
-
-|Java only
-
 | xref:implementing-portable-serialization.adoc[`Portable`]
+*Deprecated: Use Compact Serialization. This will be removed as of version 6.0*
```

Comment:
Can we turn this into a admonition or warning? Not familiar enough with the syntax yet but drawing attention to it with colour/symbol etc. would help.

I'd go for a "Hazelcast strongly recommends using Compact Serialization as Portable will be deprecated and removed in version 6.0" or similar...